resources/definitions/seckit_sktank.def.json

```
@@ -0,0 +1,59 @@
+{
+    "version": 2,
+    "name": "SecKit SK-Tank",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Ernest Lin",
+        "manufacturer": "SecKit 3DP Design",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "has_machine_quality": false,
+        "preferred_quality_type": "normal",
+        "machine_extruder_trains":
+        {
+            "0": "seckit_sktank_extruder_0"
+        }
+    },
+
+
+    "overrides": {
+        "machine_name": { "default_value": "SecKit SK-Tank" },
+        "machine_width": {
+            "default_value": 350
+        },
+        "machine_depth": {
+            "default_value": 350
+        },
+        "machine_height": {
+            "default_value": 400
+        },
+        "machine_center_is_zero": {
+            "default_value": false
+        },
+		"machine_heated_bed": {
+			"default_value": true
+		},
```

Use spaces instead of tabs
```suggestion
        "machine_heated_bed": {
            "default_value": true
        },
```